developer_setup/classic_ui_split.md

```diff
@@ -14,7 +14,7 @@ But if you do..
 1. `git clone git@github.com:YOU/manageiq-ui-classic`
 1. `cd manageiq-ui-classic`
 1. `git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic.git`
-1. `ln -s ../../manageiq spec/`
+1. `ln -s ../manageiq spec/`
```

Comment:
@isimluk I'm not sure what you are saying.  @chessbyte actually tried the directions as listed and they don't work.  The only thing that worked was his change above.